packages/tree-widget/src/components/trees/ClassificationsTree.tsx

```diff
@@ -0,0 +1,39 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) 2019 - present Bentley Systems, Incorporated. All rights reserved.
+ * Licensed under the MIT License. See LICENSE.md in the project root for license terms.
+ *--------------------------------------------------------------------------------------------*/
+import * as React from "react";
+import { Ruleset } from "@bentley/presentation-common";
+import { SimpleTreeWithRuleset } from "./TreeWithRuleset";
+import { IModelConnection } from "@bentley/imodeljs-frontend";
+import { PresentationTreeDataProvider } from "@bentley/presentation-components";
+import classificationRules from "../rulesets/ClassificationSystems.json";
+import { connectIModelConnection } from "@bentley/ui-framework";
+
+export interface ClassificationsTreeProps {
+  iModel: IModelConnection;
+}
+
+export class ClassificationsTree extends React.Component<
```

Comment:
Would be nice if this was a functional component. It would also keep it consistent with the other Trees. But this probably shouldn't block your PR.